app/forms/questionnaire_form.py

```diff
@@ -283,13 +291,19 @@ def _validate_calculated_question(
         question: QuestionSchemaType,
         target_total: Any,
         currency: Optional[str],
+        decimal_places: int | None,
     ) -> bool:
         messages = None
         if "validation" in question:
             messages = question["validation"].get("messages")
 
         validator = SumCheck(messages=messages, currency=currency)
 
+        if decimal_places:
+            formatted_total = Decimal(format(target_total, f".{decimal_places}f"))
```

Comment:
I don't follow why this is being done. Can you clarify.

Reply:
Forgot to remove this and pass it to the method, fixed now 👍